frontend/src/components/layout/Navigation.tsx

```
@@ -0,0 +1,123 @@
+    href: "/time-tracking",
+    icon: ClockIcon,
+  },
+  {
+
+    name: "Tasks",
+    href: "/tasks",
+    icon: ClipboardDocumentListIcon,
+  },
+  {
+
+    name: "Profile",
```

Biome (2.5.3)

[error] This property is later overwritten by an object member with the same name.

(lint/suspicious/noDuplicateObjectKeys)